src/main/java/brave/opentracing/BraveSpanBuilder.java

```diff
@@ -119,17 +120,37 @@
     @Override
     public BraveSpan start() {
         boolean server = Tags.SPAN_KIND_SERVER.equals(tags.get(Tags.SPAN_KIND.getKey()));
+        // adjust sampling decision
+        // https://github.com/openzipkin/brave/tree/master/brave#sampling
+        String sampling = tags.get(Tags.SAMPLING_PRIORITY.getKey());
+        Boolean notSampled = null;
```

Comment:
I think it would be easier to understand and most congruent if we limited the implementation to affecting sampling decision on a new trace.

Reseting sampling on an existing trace doesn't fit with the "before the fact" policy that's status quo here.